src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+    private final String color;
+
+    public Circle(double radius, String color) {
+        this.radius = radius;
+        this.color = color;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
+
+    public void draw() {
+        System.out.printf("Figure: %s, Area: %f sq. units, Radius: %f units, Color: %s\n",

```

Comment:
`println()` will make the output much clearer as each entry will be printed in separate line

Reply:
I stuck with printf, but used a multiline string to achieve the same effect and improve readability of both code and output